julia-mode.el

```diff
@@ -50,6 +50,13 @@
   :type 'integer
   :group 'julia)
 
+(defcustom julia-force-tab-complete t
+  "Use Tab for completion instead of M-Tab in `julia-mode'.
+This overrides `tab-always-indent' in `julia-mode' buffers. It also
+enables `abbrev-mode' so that inserting a space character will replace
+a LaTeX string with a unicode symbol."
+  :type 'boolean)
+
```

Comment:
If you want to `expand-abbrevs` after any punctuation, eg. \\alpha* => {laxex  alpha}*, you could add 
a local `post-self-insert-hook` along the lines of

```
(defun julia-abbrev-expand-after-symbols-hook ()
  (and (memq (car (syntax-after (1- (point)))) '(0 3))
       (setq this-command 'julia-abbrev-expand-after-symbols)
       (expand-abbrev)))
```
which I use in, eg. elisp, where I want my abbrevs to expand after I enter a ')' for example.
It could be another user choice to consider.  IIRC I needed to add this as a `post-self-insert` command because abbrev expansion after word syntax was hard-coded in C, but the implementation may have changed.

Reply:
The problem with this is all of the LaTeX codes that include punctuation (or even parentheses). Some examples: `\^)`, `\_=`, `\:soccer:`, `\^+`. I guess in practice, as long as none of the LaTeX codes that include symbols have other LaTeX codes that are superstrings of them cut off at the symbol it could be fine.

Reply:
> I don't think converting \sqrt to √ without further input is a good solution, cause you might still be planning to write something else

Then this may just be my personal preference --- if I want to choose between `\sqrt` and `\sqrtbottom`, I would complete on `\sqr` and pick from the popup. If feasible, keeping this modular and customizable would be nice.

Reply:
>  Some examples: \^), \_=, \:soccer:, \^+

Good point -- the above is too simple for this case